README.md

```diff
@@ -1,13 +1,18 @@
 # Project Todos
 
-Replace this readme with your own information about your project.
+This week we built a ToDo app were you can add tasks, delete tasks and toggle whether a task is done or not and change it back to not done. 
+Also it shows a count of the done tasks.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week was our second with Redux and we practiced building our ToDo app from scratch creating the slice with dummy data and then uniqid for new ToDo tasks. 
+We used reducers, useState, useSelectors and useDispatch to send and recieve data and updates from the store, and splice and push to add and delete tasks.
+We continued with styled components and this week was no given design but our own, and I started the project by using a FigJam board for structure.   
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I struggled with positioning the form, it's design and size and the responsiveness. I solved this by using vW and rem on the width.
+I solved an issue with text overflowing by disabeling the button for text > 25 characters and writing a placeholder that the task should be short.
+If I had more time I would find a more elegant solution to this. Also add todays date, style the checkbox checked color to not be blue but suit my colors more and remove the examlpe ToDo task.    
 
-## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+## View it live
+https://lovely-todo-list.netlify.app/
```

Comment:
Very well written and nice readme! Interesting and inspiring to hear about your solutions and approaches!